src/components/FunctionButton.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+export default function FunctionButton(props) {
```

Comment:
We can also write it like this:

const  FunctionButton = (props) => <button
      class="text-white bg-gradient-to-r from-blue-500 via-blue-600 to-blue-700 hover:bg-gradient-to-br focus:ring-4 focus:outline-none focus:ring-blue-300 dark:focus:ring-blue-800 font-medium rounded-lg text-sm px-5 py-2.5 text-center me-2 mb-2"
      type="button"
      onClick={props.func}
    >
      {props.text}
    </button>
    
    export default FunctionButton;